jipdate.py

```diff
@@ -191,7 +196,7 @@ def get_jira_issues(jira, exclude_stories, epics_only, all_status, filename,
         f.write("# Header: %s\n" % issue.fields.summary)
         f.write("# Type: %s\n" % issue.fields.issuetype)
         f.write("# Status: %s\n" % issue.fields.status)
-        f.write("No updates since last week.\n\n")
+        f.write("# No updates since last week.\n\n")
```

Comment:
I do agree with you about not being very useful, but again, we're asked to update every card on weekly basis and if you don't add anything, well, then we are not doing what we are being asked to do. Personally I've always said to people, if there are no (recent) updates in a card, then it is the same status as the last update.

Adding a non-useful message like this will eventually open up the eyes for (some) people making it clear that doing weekly updates might not be that useful in the end. I.e, part if this is to prove that weekly updates are not always what we want. Counterproductive? Absolutely ...

As you said, I'll make that configurable and then it will be up to each person to decide what should be there by default or not.